WalletWasabi/Blockchain/Analysis/BlockchainAnalyzer.cs

```diff
@@ -135,10 +135,11 @@ private void AnalyzeCoinjoinWalletOutputs(SmartTransaction tx, double startingMi
 			// Picking randomly an output would make our anonset: total/ours.
 			double anonymityGain = Math.Min(CoinjoinAnalyzer.ComputeAnonymityContribution(virtualOutput.Coins.First()), foreignInputCount);
```

Comment:
For better readability consider moving this code down where `anonymityGain` is used.